src/hw4_Polymorphism/Figures/Triangle.java

```
@@ -0,0 +1,26 @@
+package hw4_Polymorphism.Figures;
+
+public class Triangle implements Figure{
+    private final int sideA;
+    private final int sideB;
+    private final int sideC;
+
+    public Triangle(int sideA, int sideB, int sideC) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.sideC = sideC;
+        if(!isTriangle()) System.out.println("This is not a triangle");
```

I like the validation, but it's better to throw an error. Try to avoid using sout in the code. Consider using logger [https://www.digitalocean.com/community/tutorials/logger-in-java-logging-example](https://www.digitalocean.com/community/tutorials/logger-in-java-logging-example)